src/AddItem.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import firebase from 'firebase';
-
+import NavLinks from './NavLinks';
```

Comment:
I'm not sure where all you intend for the `NavLinks` to be rendered, but if you want them on all views, I might move this up to `App,js` so that you only have to import and use it one time. 

That kind of thing would also help you preserve state if at some point down the line you had some display logic in the navigation. We're not doing that in this app, but in the future if you wanted to keep going with it you totally could!

Reply:
The links only render on list and addItem views. They should not be seen on the welcome/home page.

Reply:
Yeah we were thinking that it would look 'cleaner' if they only rendered on the list and addItem views